### DISCUSSION TITLE
Sometimes airflow keeps throwing Max retries exceeded with url: /api/v1/dags/<dag-id>/dagRuns

### DISCUSSION THREAD

**Discussion by faddiekh** [apache/airflow#20917] [Q&A] [answered]
I have no idea why but sometimes airflow keeps throwing following error even though there's just one request at a time.
`HTTPSConnectionPool(host='domain', port=443): Max retries exceeded with url: /api/v1/dags/<dag-id>/dagRuns (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbdf8e5b580>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))`

airflow docker version: bitnami/airflow:2.2.3-debian-10-r16
deployed on aks

**Comment by potiuk** [comment, marked as answer]
This is the problem in DNS in your AKS instance. It's not specific to airlfow and it should be raised to AKS support.

**Reply by faddiekh** [reply]
Looks like the issue is also not from aks, when I'm triggering dag from locally it's getting triggered fine but whenever I'm triggering though azure functions I'm getting this error. Thanks for your direction though. cheers

**Reply by potiuk** [reply]
You might check this guide: https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-name-resolution-for-vms-and-role-instances
